<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QWidget" name="container" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/icons/background.png);
border: 1px solid grey</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_Save_As">
     <property name="title">
      <string>&amp;Save As</string>
     </property>
    </widget>
    <addaction name="menu_Save_As"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionPen_color"/>
    <addaction name="actionPen_width"/>
    <addaction name="actionClear_screen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBrush"/>
    <addaction name="actionBucket"/>
    <addaction name="actionEraser"/>
    <addaction name="actionSelect"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionSpray"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionBrightness"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuFilters"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionClear_screen"/>
   <addaction name="actionBrush"/>
   <addaction name="actionBucket"/>
   <addaction name="actionEraser"/>
   <addaction name="actionSpray"/>
   <addaction name="separator"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionRectangle"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSelect"/>
   <addaction name="separator"/>
   <addaction name="actionColorpicker"/>
   <addaction name="actionPen_color"/>
  </widget>
  <widget class="QDockWidget" name="rightDockWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="docker">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="layerList">
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>152</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="removeLayerButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="addLayerButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPen_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/wheel.png</normaloff>:/icons/wheel.png</iconset>
   </property>
   <property name="text">
    <string>Pen color</string>
   </property>
  </action>
  <action name="actionPen_width">
   <property name="text">
    <string>Pen width</string>
   </property>
  </action>
  <action name="actionClear_screen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Clear screen</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/repeat.png</normaloff>:/icons/repeat.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/brush.png</normaloff>:/icons/brush.png</iconset>
   </property>
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
  <action name="actionBucket">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/fill.png</normaloff>:/icons/fill.png</iconset>
   </property>
   <property name="text">
    <string>Bucket fill</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/erase.png</normaloff>:/icons/erase.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rectangle.png</normaloff>:/icons/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/oval.png</normaloff>:/icons/oval.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionSpray">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/spray.png</normaloff>:/icons/spray.png</iconset>
   </property>
   <property name="text">
    <string>Spray</string>
   </property>
  </action>
  <action name="actionColorpicker">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/picker.png</normaloff>:/icons/picker.png</iconset>
   </property>
   <property name="text">
    <string>Colorpicker</string>
   </property>
  </action>
  <action name="actionAddLayer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Layer</string>
   </property>
  </action>
  <action name="actionRemoveLayer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove Layer</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
